scaling_performance/optimizing_storage.adoc

```diff
@@ -0,0 +1,114 @@
+[[scaling-performance-optimizing-storage]]
+= Optimizing Storage
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+
+
+== Optimizing Storage
+
+Docker stores images and containers in a graph driver (a pluggable storage
+backend), such as Device Mapper, OverlayFS, and Btrfs.
+
+Each have advantages and disadvantages. For example, OverlayFS is faster than
+Device Mapper at starting and stopping containers, but is not POSIX compliant,
+and does not yet support SELinux. Red Hat is working to add SELinux support for
+OverlayFS, but many of the POSIX compliance issues are architectural limitations
+of a union file system. For more information about OverlayFS, including
+supportability and usage caveats,
+link:https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html-single/7.2_Release_Notes/index.html#technology-preview-file_systems[see
+the RHEL 7.2 Release Notes].
+
+In production environments, using a LVM thin pool on top of regular block
+devices (not loop devices) for container images and container root file systems
+storage is recommended. To ease Docker backend storage configuration, use the
+`docker-storage-setup` utility, which automates much of the configuration
+details:
+
```

Comment:
There are known performance issues with using the loop device backend.  Docker will log a warning message if it is started with the loop device but will still allow the user to continue.

The error message a user would see if they are using loopback devices is:
```
devmapper: Usage of loopback devices is strongly discouraged for production use. Please use `--storage-opt dm.thinpooldev` or use `man docker` to refer to dm.thinpooldev section.```